pihole-google.txt

```diff
@@ -7352,4 +7352,332 @@ www.widgets.ft.nest.com
+mimageads2.googleadservices.com
+mimageads3.googleadservices.com
+mimageads4.googleadservices.com
+mimageads5.googleadservices.com
+mimageads6.googleadservices.com
+mimageads7.googleadservices.com
+mimageads8.googleadservices.com
+mimageads9.googleadservices.com
+adsense.google.com
+www.googleoptimize.com
+choices.consentframework.com
+s.skimresources.com
+pixel.wp.com
+docs.google.com
+analytics.l.google.com
+clouderrorreporting.googleapis.com
+googlecode.com
+doubleclickbygoogle.com
+www.googletagservices.com
+www.googleoptimize.com
+cse.google.com
+video-stats.video.google.com
+clientmetrics-pa.googleapis.com
+mail-ads.google.com
+mimageads.googleadservices.com
+mimageads1.googleadservices.com
+mimageads2.googleadservices.com
+mimageads3.googleadservices.com
+mimageads4.googleadservices.com
+mimageads5.googleadservices.com
+mimageads6.googleadservices.com
+mimageads7.googleadservices.com
+mimageads8.googleadservices.com
+mimageads9.googleadservices.com
+adsense.google.com
+www.googleoptimize.com
+tpc.googlesyndication.com
+google-analytics.com
+www.google-analytics.com
+analytics.withgoogle.com
+googleanalytics.com
+ssl.google-analytics.com
+www-google-analytics.l.google.com
+ssl-google-analytics.l.google.com
+google-analytics.l.google.com
+g.doubleclick.net
+securepubads.g.doubleclick.net
+displayads-formats.googleusercontent.com
+pagead46.l.doubleclick.net
+partnerad.l.doubleclick.net
+buttons.googlesyndication.com
+pagead1.googlesyndication.com
+pagead2.googlesyndication.com
+pagead3.googlesyndication.com
+pagead.googlesyndication.com
+googleads2.g.doubleclick.net
+googleads6.g.doubleclick.net
+googleads7.g.doubleclick.net
+googleads4.g.doubleclick.net
+2542116.fls.doubleclick.net
+9309168.fls.doubleclick.net
+googleads.g.doubleclick.net
+static.doubleclick.net
+cm.g.doubleclick.net
+pubads.g.doubleclick.net
+www.googletagservices.com
+adservice.google.com
+adservice.io
+googleadservices.com
+feedads.googleadservices.com
+partner.googleadservices.com
+www.googleadservices.com
+9098b63249ee4522c5d8677059fe22e4.safeframe.googlesyndication.com
+pagead-googlehosted.l.google.com
+firebase.com
+firestore.googleapis.com
+firebaseremoteconfig.googleapis.com
+firebaseinstallations.googleapis.com
+firebaselogging.googleapis.com
+firebase-settings.crashlytics.com
+try.crashlytics.com
+settings.crashlytics.com
+crashlytics.l.google.com
+crashlytics.com
+doubleclick-net.com
+doubleclickbygoogle.com
+googleapi.club
+googletagservices.com
+imasdk.googleapis.com
+googletrackmanager.compartner.googleadservices.com
+partner.video.syndication.msn.com
+googleadapis.l.google.com
+googleads.github.io
+gstaticadssl.l.google.com
+pagead2.googlesyndication.com
+mail-ads.google.com
+tpc.googlesyndication.com
+g.doubleclick.net
+imasdk.googleapis.com
+pagead2.googlesyndication.com
+partner.googleadservices.com
+googleoptimize.com
+googletagmanager.com
+google-analytics.com
+googleadservices.com
+googlerank.info
+www-google-analytics.l.google.com
```

Comment:
```suggestion

```

Reply:
Done

Reply:
Are you sure that you commit your changes?

Reply:
Now??

Reply:
The changes should be made